### PR TITLE
koord-scheduler: optimize ElasticQuota plugin's update logic

### PR DESCRIPTION
**Pull request by xulinfei1996**
Signed-off-by: xulinfei.xlf <xulinfei.xlf@alibaba-inc.com>

### Ⅱ. Does this pull request fix one issue?

fixes https://github.com/koordinator-sh/koordinator/issues/657
### V. Checklist

- [ ] I have written necessary docs and comments
- [ ] I have added necessary unit tests and integration tests
- [ ] All checks passed in `make test`

### CODE REVIEW

File: pkg/scheduler/plugins/elasticquota/controller.go [hunk 1]
```diff
@@ -98,7 +98,7 @@ func (ctrl *Controller) syncHandler() []error {
 			}
 
 			// Ignore this loop if the runtime/request/used doesn't change
-			if quotav1.Equals(eq.Status.Used, used) &&
+			if (quotav1.Equals(eq.Status.Used, used) || (quotav1.IsZero(eq.Status.Used) && quotav1.IsZero(used))) &&
 				eq.Annotations[extension.AnnotationRuntime] == runtime &&
```
[the remark this answers is not in the document]
Comment (xulinfei1996) [reply pkg/scheduler/plugins/elasticquota/controller.go]: fixed

**Review by eahydra — APPROVED** [review]
/lgtm

File: pkg/scheduler/plugins/elasticquota/core/group_quota_manager.go [hunk 2]
```diff
@@ -513,6 +516,10 @@ func (gqm *GroupQuotaManager) updatePodUsedNoLock(quotaName string, oldPod, newP
 
 	deltaUsed := quotav1.Subtract(newPodUsed, oldPodUsed)
 	if quotav1.IsZero(deltaUsed) {
+		if newPod != nil && oldPod != nil {
```
Comment (buptcozy) [inline pkg/scheduler/plugins/elasticquota/core/group_quota_manager.go:519]: as discuessed, get pod name corrctly

Comment (xulinfei1996) [reply pkg/scheduler/plugins/elasticquota/core/group_quota_manager.go]: fixed

**Review by buptcozy — COMMENTED** [review]
/lgtm

**Review by buptcozy — APPROVED** [review]
/approve

**Review by yihuifeng — COMMENTED** [review]
/lgtm

**Review by yihuifeng — APPROVED** [review]

**Review by eahydra — APPROVED** [review]
/approve

**Pull request merged**